chain-abci/src/enclave_bridge/edp/mod.rs

```diff
@@ -22,36 +22,55 @@ use tokio::net::{TcpListener, TcpStream};
 
 use std::os::unix::net::UnixStream;
 
+const REMOTE_ATTESTATION_PROXY: &str = "ra-sp-server";
+
 /// pair of unix domain sockets
 /// enclave_stream is only needed / passed in `connect_stream`
 /// `runner_stream` is shared in chain-abci app
 #[derive(Debug)]
 pub struct TxValidationApp {
     enclave_stream: Option<UnixStream>,
     runner_stream: Arc<Mutex<UnixStream>>,
+    /// `ra-sp-server` address for remote attestation. E.g. `0.0.0.0:8989`
+    /// FIXME: enclave direct connection -- not via TCP proxy
+    sp_address: Option<String>,
+    tdbe_stream: Option<UnixStream>,
```

Comment:
Why is this an `Option`? When will this be `None`?

Reply:
they'll be None in the instances on the chain-abci side